neurotoxic-game-threat-model.md

```
@@ -0,0 +1,97 @@
+  - **Validation:** MIDI duration and note scheduling guards in audio engine before playback. (Runtime skips invalid notes and validates durations.)【F:src/utils/audioEngine.js†L792-L900】
+
+#### Diagram
+```mermaid
+flowchart TD
+  A["Player Browser"] --> B["React SPA"]
+  B --> C["localStorage"]
+  B --> D["Static Assets"]
+```
+
+## Assets and security objectives
+| Asset | Why it matters | Security objective (C/I/A) |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)